django/apps/registry.py

```diff
@@ -329,5 +333,43 @@ def clear_cache(self):
             for model in self.get_models(include_auto_created=True):
                 model._meta._expire_cache()
 
+    def lazy_model_operation(self, function, *model_keys):
+        """
+        Take a function and a number of ("app_label", "ModelName") tuples, and
+        when all the corresponding models have been imported and registered,
+        call the function with the model classes as its arguments.
+
+        The function passed to this method must accept exactly n models as
+        arguments, where n=len(model_keys).
+        """
+
+        # If this function depends on more than one model, we recursively turn
+        # it into a chain of functions that accept a single model argument and
+        # pass each in turn to lazy_model_operation.
+        model_key, more_models = model_keys[0], model_keys[1:]
```

Comment:
I think I'd rather keep it as is. There's nothing special about the first argument to this method (unlike in ``lazy_related_operation``) and I'd prefer its signature to reflect that than to save a line here.